packages/vagov-eslint/package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "vagov-eslint",
+    "description": "Shared JS linting package",
+    "version": "1.0.0",
+    "bin": {
+        "vagov-eslint": "./bin/vagov-eslint"
+    },
+    "dependencies": {
+        "eslint": "4.4.1",
+        "eslint-config-airbnb": "15.1",
+        "eslint-config-prettier": "^4.1.0",
+        "eslint-plugin-import": "^2.9.0",
+        "eslint-plugin-jest": "^22.3.0",
+        "eslint-plugin-jsx-a11y": "5.1",
+        "eslint-plugin-mocha": "^5.3.0",
+        "eslint-plugin-prettier": "^3.0.1",
+        "eslint-plugin-react": "7.2.1",
+        "eslint-plugin-va-enzyme": "file:./custom-eslint/va-enzyme",
+        "prettier": "^1.16.4"
+    },
+    "scripts": {
+        "postinstall" : "cp .eslintrc $INIT_CWD && cp .eslintignore $INIT_CWD"
```

Comment:
`postinstall` is executed relative to the node module. `$INIT_CWD` is a reference to the directory where the install script is called. 

This copies the eslint configs making them available to the `npx` commands.